### PR TITLE
Added button for back to top - https://github.com/fossasia/gci17.fossasia.org/issues/422 - as this.

### PR DESCRIPTION
**Pull request by supunTE**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request
I added a back to up floating button as this issue https://github.com/fossasia/gci17.fossasia.org/issues/422. 
https://stedirisuriya.github.io/gci17.fossasia.org/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,116 +1,133 @@
-# FOSSASIA GCI 2017 Site
```
Comment (supunTE) [inline README.md]: Came with my last PR commit when merging.

File: README.md [hunk 2]
```diff
@@ -1,116 +1,133 @@
...
-{% include footer.html %}
-```
-* Add a link in the nav bar (in `_includes/header.html`)
-
-## Installation Steps:
-- [Local Setup on Windows](/installation_windows.md)
-- [Local Setup on Linux](/installation_linux.md)
-- [Local Setup on MacOS](/installation_macos.md)
\ No newline at end of file
+# FOSSASIA GCI 2017 Site
```
Comment (supunTE) [inline README.md]: This also the same.

File: _data/blogs.yml [hunk 3]
```diff
@@ -182,3 +182,13 @@
   title: Interview with two active Open Source Contributors
```
Comment (supunTE) [inline _data/blogs.yml]: Came with another commits by other when merging

File: _data/mentors.yml [hunk 4]
```diff
@@ -255,7 +255,6 @@
   lat: 13.405
   lng: 52.52
 
-
 - name: Ayush Gupta
```
Comment (supunTE) [inline _data/mentors.yml]: When merging

File: _data/students.yml [hunk 5]
```diff
@@ -157,6 +151,15 @@
   lat: 13.756331
   lng: 100.501765
 
+- name: Piyush Raj
```
Comment (supunTE) [inline _data/students.yml]: When merging

File: _data/video.yml [hunk 6]
```diff
@@ -13,3 +13,9 @@
   title: Interview - GCI - Fossasia - Task (Blog)
   img: Beta_KIng.jpg
   author_name: Yash Kothari
+  
+- url: https://www.youtube.com/watch?v=6IOoht-49Yc
```
Comment (supunTE) [inline _data/video.yml]: when merging

File: _includes/footer.html [hunk 7]
```diff
@@ -157,16 +157,6 @@ <h6 class="title">Instagram</h6>
 										<i class=" ti-instagram" aria-hidden="true"></i>
 									</a>
 								</li>
-								<li>
```
Comment (supunTE) [inline _includes/footer.html]: When merging

File: _includes/footer.html [hunk 8]
```diff
@@ -157,16 +157,6 @@ <h6 class="title">Instagram</h6>
...
-									<a href="https://dribbble.com/fossasia">
```
Comment (supunTE) [inline _includes/footer.html]: When merging

File: _includes/footer.html [hunk 9]
```diff
@@ -191,8 +181,8 @@ <h6 class="title">Instagram</h6>
 		<script async defer src="https://apis.google.com/js/platform.js" ></script>
 		<script src="js/contributors.js"></script>
 		<script src="js/inview.js"></script>
+		<script src="js/backtop.js"></script>
```
Comment (supunTE) [inline _includes/footer.html]: This is for connect js that containing Back to top files with html file

File: _includes/header.html [hunk 10]
```diff
@@ -17,6 +17,7 @@
 		<link href="css/custom.css" rel="stylesheet" type="text/css" media="all" />
 		<link href="css/blog.css" rel="stylesheet" type="text/css" media="all" />	
 		<link href="css/hover-media-links.css" rel="stylesheet" type="text/css" media="all">
+		<link href="css/backtop.css" rel="stylesheet" type="text/css" media="all">
```
Comment (supunTE) [inline _includes/header.html]: This connect back top top button's css with html file

File: index.html [hunk 11]
```diff
@@ -2,6 +2,8 @@
 ---
 {% include header.html %}
```
Comment (supunTE) [inline index.html]: when merging

File: installation/installation_linux(sin).md [hunk 12]
```diff
@@ -0,0 +1,21 @@
+# ප්‍රාදේශීය සංවර්ධන සැකසුම
```
Comment (supunTE) [inline installation/installation_linux(sin).md]: From my last PRs commit when merging

File: installation/installation_linux.md [hunk 13]
```diff
@@ -4,16 +4,17 @@ The instructions on this page will guide you in setting up a local development
 environment in Linux.
 
 ### Ruby Installation
-1. Open a Terminal and use a package manager to install ruby. For example, on Ubuntu or Debian use `sudo apt-get install ruby` or on Arch Linux `sudo pacman -S ruby`
```
Comment (supunTE) [inline installation/installation_linux.md]: Last commit of mine

File: installation/installation_macos(sin).md [hunk 14]
```diff
@@ -0,0 +1,27 @@
+# ප්‍රාදේශීය සංවර්ධන සැකසුම
```
Comment (supunTE) [inline installation/installation_macos(sin).md]: When merging

File: installation/installation_macos(sin).md [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+
+මෙම පිටුවෙහි ඇති උපදෙස් ඔබට MacOS පාරිසරිකය තුල ප්‍රාදේශීය ප්‍රවර්ධන සැකසුමට මග පෙන්වනු ඇත.
+
+### Apple සංවර්ධන කට්ටලය ස්ථාපනය කිරීම
```
Comment (supunTE) [inline installation/installation_macos(sin).md]: When merging

File: installation/installation_macos.md [hunk 16]
```diff
@@ -4,23 +4,23 @@ The instructions on this page will guide you in setting up a local development
 environment in MacOS.
 
 ### Apple Developer Tools Installation
-1. On OS X, open your Applications folder, then open the Utilities folder. Open the Terminal application.
```
Comment (supunTE) [inline installation/installation_macos.md]: When merging

File: installation/installation_windows(sin).md [hunk 17]
```diff
@@ -0,0 +1,26 @@
+# ප්‍රාදේශීය සංවර්ධන සැකසුම
```
Comment (supunTE) [inline installation/installation_windows(sin).md]: When merging

File: installation/installation_windows.md [hunk 18]
```diff
@@ -8,7 +8,7 @@ environment in Windows OS.
 e.g. for 64-bit architecture install: [Ruby2.3.3](https://dl.bintray.com/oneclick/rubyinstaller/ruby-2.3.3-i386-mingw32.7z)
 2. [Install corresponding Ruby DevKit](https://rubyinstaller.org/downloads/) e.g. corresponding [developer kit](https://dl.bintray.com/oneclick/rubyinstaller/DevKit-mingw64-64-4.7.2-20130224-1432-sfx.exe).
```
Comment (supunTE) [inline installation/installation_windows.md]: When merging

**Review by amryarios — APPROVED** [review]
Good Work

**Review by dilpreetsio — COMMENTED** [review]
@STEdirisuriya Please resolve conflicts

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
why you always mess with your PR. As usual your repo is not up-to-date. Please update yuor forked repo before starting to work. Thanks

File: css/backtop.css [hunk 19]
```diff
@@ -0,0 +1,66 @@
+#scrolltop {
```
Comment (vaibhavsingh97) [inline css/backtop.css]: Please indent your file

File: index.html [hunk 20]
```diff
@@ -2,6 +2,8 @@
...
 
+<a href="javascript:void(0);" id="rocketmeluncur" class="showrocket" ><i></i></a>
```
Comment (vaibhavsingh97) [inline index.html]: why dummy i tag  `<i></i>` is present ?

File: js/backtop.js [hunk 21]
```diff
@@ -0,0 +1,34 @@
+jQuery(window).scroll(function(){
```
Comment (vaibhavsingh97) [inline js/backtop.js]: Please indent your file

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

File: index.html (hunk 20 above)
Comment (supunTE) [reply index.html]: Can't you see that `<i>`  tag has a command in css file. If it's not there some parts of animation are missing!

File: css/backtop.css (hunk 19 above)
Comment (supunTE) [reply css/backtop.css]: I'll indent

Comment (supunTE) [reply css/backtop.css]: I indent it

**Review by NaviRocker — CHANGES_REQUESTED** [review]
I think you should update your Repo.

**Comment by supunTE** [comment]
If I update my repo there will came lot of commit .
Didn't?

File: README.md [hunk 22]
```diff
@@ -130,5 +130,4 @@ permalink: <some permalink>
     - [In Sinhala - සිංහලෙන්](/installation/installation_linux(sin).md) 
 - Local Setup on MacOS 
     - [In English](/installation/installation_macos.md)
-    - [In Sinhala - සිංහලෙන්](/installation/installation_macos(sin).md) 
-
+    - [In Sinhala - සිංහලෙන්](/installation/installation_macos(sin).md)
```
Comment (vaibhavsingh97) [inline README.md]: extraneous change. Please add one blank line

File: _includes/footer.html (hunk 9 above)
Comment (vaibhavsingh97) [reply _includes/footer.html]: Why are you making new js file? You can simple append the code in `script.js`

File: _includes/header.html (hunk 10 above)
Comment (vaibhavsingh97) [reply _includes/header.html]: Same

File: index.html (hunk 20 above)
Comment (vaibhavsingh97) [reply index.html]: bette use `id` istead of dummy tags

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

File: README.md (hunk 22 above)
Comment (supunTE) [reply README.md]: I said it is a past one

File: README.md [hunk 23]
```diff
@@ -134,4 +134,3 @@ permalink: <some permalink>
     - [In English](/installation/installation_macos.md)
     - [In Sinhala - සිංහලෙන්](/installation/installation_macos(sin).md) 
     - [In Polish - Polski](/installation/installation_macos(pl).md)
-
```
Comment (akshatnitd) [inline README.md]: Why did you remove this line. Add it back

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Unwanted changes have come in this PR. Rebase properly

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@STEdirisuriya  Please **follow best practices** as your **fork is not updated** as usual and there are lots of previous commits, close this PR and create a new one with **proper commit and PR message**.
